manager/main.go

```diff
@@ -182,15 +182,13 @@ func main() {
 	}
 }
 
-// init vault client and mount the base directory for storing credentials
+// init vault client
+// TODO: the vault client should be used to add roles for the modules.
 func initVaultConnection() (vault.Interface, error) {
```

Comment:
In this implementation manager does not communicate with vault at all. This means:
1. No need for VAULT_TOKEN passed to the manager pod
2. No need for vault.Interface inside manager

Reply:
I left it as is as the next step would be to let the manager to set Vault policies and Vault roles for modules.